### PR TITLE
deactivated link to followers and following when they're 0

### PR DESCRIPTION
**Pull request by shubhangi013**
Signed-off-by: shubhangi013 <choudharyshubhangi13@gmail.com>

Deactivated the link to followers and following when either of them or both of them is 0

video:

https://www.awesomescreenshot.com/video/7828842?key=390679b307ecc7bacabeeedd44ea3ccf

Fixes part of #10730 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -69,9 +69,14 @@
 
   <div>
     <i class="fas fa-user-friends"></i>
+    <%if (@profile_user.followers.count>0)%>
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:72]: @shubhangi013 The if statements don’t need brackets around them. Also add spaces after `<%` and before the `%>`.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/users/profile.html.erb [hunk 2]
```diff
@@ -69,9 +69,14 @@
...
     <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
     &#8226;
     <a href="/people/<%= @profile_user.username %>/following" class="link-secondary"><%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%></a>
+    <%else%>
+    <%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%>
+    <%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%>
+    <%end%>
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:79]: Indent the code inside the if and else statements.

File: app/views/users/profile.html.erb (hunk 1 above)
Comment (shubhangi013) [reply app/views/users/profile.html.erb:72]: Will do that! Thanks

**Comment by shubhangi013** [comment]
@TildaDares I've done all the changes.

File: package.json [hunk 3]
```diff
@@ -66,10 +66,12 @@
     "typeahead.js": "^0.11.1",
     "typeahead.js-browserify": "Javier-Rotelli/typeahead.js-browserify#~1.0.7",
     "urlhash": "^0.1.3",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^3.3.12",
     "woofmark": "~4.2.0"
   },
   "devDependencies": {
-    "webpack-dev-server": "^4.7.4"
+    "webpack-dev-server": "^3"
```
Comment (TildaDares) [inline package.json:74]: This file and `yarn.lock` shouldn't be in the PR.

File: app/views/users/profile.html.erb [hunk 4]
```diff
@@ -69,9 +69,14 @@
 
   <div>
     <i class="fas fa-user-friends"></i>
-    <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
-    &#8226;
-    <a href="/people/<%= @profile_user.username %>/following" class="link-secondary"><%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%></a>
+    <% if @profile_user.followers.count>0 %>
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:72]: This only checks if the number of followers is greater than 0 but doesn't check the following count. You need if-else statements for both `following` and `followers`. Something like this:
```
<% if @profile_user.followers.count > 0 %>
    <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
<% else %>
    <span class="text-secondary">0 followers</span>
<% end %>

```

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (shubhangi013) [reply package.json:74]: > This file and `yarn.lock` shouldn't be in the PR.

So, I should revert the changes I did to these right?

**Comment by shubhangi013** [comment]
@TildaDares could you review this once

File: app/views/users/profile.html.erb [hunk 5]
```diff
@@ -69,9 +69,17 @@
...
+      <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
+      &#8226;
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:74]: Line 74 should be on 78. Apart from that everything looks good. Great job!

Comment (shubhangi013) [reply app/views/users/profile.html.erb]: > Line 74 should be on 78. Apart from that everything looks good. Great job!

Thanks! I've done the change.

**Review by TildaDares — APPROVED** [review]
Everything looks good to me @shubhangi013! Great job.

**Comment by shubhangi013** [comment]
> Everything looks good to me @shubhangi013! Great job.

Thankyou for being patient with me! Look forward to contributing more :smile:

**Pull request merged**